src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -1006,7 +1006,8 @@ public static function standardize($strString, $blnPreserveUppercase=false)
 		$strString = Utf8::toAscii($strString);
 		$strString = preg_replace($arrSearch, $arrReplace, $strString);
 
-		if (is_numeric(substr($strString, 0, 1)))
+		// Prefix numeric values (see #707)
+		if (is_numeric($strString))
```

Comment:
Imho this is a BC break because the `StringUtil` (as the name implies) is a general util class and belongs to the public API. This is a behavioural change which I'm not comfortable with. I would prefer adding a third argument to trigger this case explicitly and make use of it when generating page aliases and html id attributes.